config/a8c-for-agencies-stage.json

```diff
@@ -52,7 +53,7 @@
 		"a8c-for-agencies-signup": true,
 		"a8c-for-agencies-referrals": true,
 		"a8c-for-agencies-settings": false,
-		"a8c-for-agencies-partner-directory": true,
+		"a8c-for-agencies-partner-directory": false,
```

Comment:
Should we get rid of this section flag & use the feature flags at these 2 places:

https://github.com/Automattic/wp-calypso/blob/58e0a8bad04668f41ccdaa3cc77f02d85f91f697/client/a8c-for-agencies/hooks/use-onboarding-tours.ts#L88

https://github.com/Automattic/wp-calypso/blob/58e0a8bad04668f41ccdaa3cc77f02d85f91f697/client/a8c-for-agencies/sections/overview/partner-directory-onboarding-card/hooks/use-partner-directory-onboard-card.ts#L21

Reply:
Good catch! Yes